config/default.yml

```diff
@@ -4,6 +4,10 @@ AllCops:
     Patterns:
     - _spec.rb
     - "(?:^|/)spec/"
+  FactoryGirl:
```

Comment:
I'm not sure where to put the option to disable the whole namespace. Should it be when it's requiring the files or a kind of verification that all factory girl cops should follow?